test/constructors.jl

```diff
@@ -28,25 +28,72 @@ module TestConstructors
     @test isequal(dt[:x1], NullableArray(dt2[:x1]))
     @test isequal(dt[:x2], NullableArray(dt2[:x2]))
 
-    @test isequal(dt, DataTable(x1 = [0.0, 0.0, 0.0],
-                                x2 = [1.0, 1.0, 1.0]))
-    @test isequal(dt, DataTable(x1 = [0.0, 0.0, 0.0],
-                                x2 = [1.0, 1.0, 1.0],
+    @test isequal(dt, DataTable(x1 = NullableCategoricalVector([0.0, 0.0, 0.0]),
+                                x2 = NullableCategoricalVector([1.0, 1.0, 1.0])))
+    @test isequal(dt, DataTable(x1 = NullableCategoricalVector([0.0, 0.0, 0.0]),
+                                x2 = NullableCategoricalVector([1.0, 1.0, 1.0]),
                                 x3 = [2.0, 2.0, 2.0])[[:x1, :x2]])
 
     dt = DataTable(Int, 2, 2)
     @test size(dt) == (2, 2)
-    @test eltypes(dt) == [Nullable{Int}, Nullable{Int}]
+    @test eltypes(dt) == [Int, Int]
 
     dt = DataTable([Int, Float64], [:x1, :x2], 2)
     @test size(dt) == (2, 2)
-    @test eltypes(dt) == [Nullable{Int}, Nullable{Float64}]
-
-    @test isequal(dt, DataTable([Int, Float64], 2))
```

Comment:
The constructor is legitimate but the test is asking if two datatables with uninitialized columns contain the same values.

by the same logic these two vectors should be equal
```julia
julia> Vector{Float64}(10)
10-element Array{Float64,1}:
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314
 2.29759e-314

julia> Vector{Float64}(10)
10-element Array{Float64,1}:
 2.29514e-314
 9.88131e-324
 2.25711e-314
 2.29432e-314
 0.0
 2.29469e-314
 2.29469e-314
 2.29514e-314
 2.29469e-314
 5.45361e-312
```